tests/test__collection_api.py

```diff
@@ -863,6 +875,23 @@ def test__create_uniq_idxs_without_ordering(self):
 
         self.assertEqual(self.db.collection.find({}).count(), 1)
 
+    @skipIf(not _HAVE_PYMONGO, "pymongo not installed")
+    def test__create_indexes_uniq_idxs(self):
+        indexes = [
+            pymongo.operations.IndexModel([('value', pymongo.ASCENDING)], unique=True),
+            pymongo.operations.IndexModel([('name', pymongo.ASCENDING)], unique=True)
+        ]
+        self.db.collection.create_indexes(indexes)
+
+        self.db.collection.insert({"value": 1, "name": "bob"})
+        # Ensure both uniq indexes have been created
```

Comment:
typo: uniq -> unique. Also please use the opportunity to end your comment with a dot.